eng/config/test/BannedSymbols.txt

```diff
@@ -0,0 +1,2 @@
+M:Moq.Mock`1.#ctor; Pass MockBehavior.Strict explicitly to the constructor for Mock<T>
+F:Moq.MockBehavior.Loose; Use MockBehavior.Strict for mocks
```

Comment:
I get the first one, but we don't want people to be opting in, even explicitly?

Reply:
No. Mocks are challenging for maintainability because they do not show up as implementations of classes and interfaces, so they are prone to unintended deviations from adherence to API post-conditions. Loose mocks are the worst case scenario because they also don't show up as _references_ to the API which changed.